ambry-store/src/test/java/com.github.ambry.store/LogTest.java

```diff
@@ -116,12 +117,13 @@ public void logAppendTest() {
       Assert.assertEquals(logTest.getLogEndOffset(), 4000);
 
       // write more and verify we fail to write
+      String str = tempFile.getParent()+".";
       try {
         logTest.appendFrom(ByteBuffer.wrap(testbuf));
         Assert.assertTrue(false);
       } catch (IllegalArgumentException e) {
         Assert.assertEquals(registry.getCounters().
-            get("com.github.ambry.store.Log." + tempFile.getParent() + "-overflowWriteError").getCount(), 1);
+            get("com.github.ambry.store.Log." + str + "OverflowWriteError").getCount(), 1);
```

Comment:
I don't think the `str` is required. Simply capitalizing the name of the metric and prefixing with a "." should do let the test succeed.